### DISCUSSION TITLE
LELLKI WP34-EU wrongly detected. how to fix?

### DISCUSSION THREAD

**Discussion by zandadoum** [Koenkk/zigbee2mqtt#12377] [Devices]
Hello,
I just received 2x LELLKI WP34-EU https://www.zigbee2mqtt.io/devices/WP34-EU.html

everything works fine, however z2m detects them as a different model LELLKI WP33-EU https://www.zigbee2mqtt.io/devices/WP33-EU.html

can i fix this somehow?

**Comment by Koenkk** [comment]
Can you provide the data/database.db entry of this device?

**Reply by zandadoum** [reply]
is this ok?
https://pastebin.com/6wan7cpm

**Comment by Koenkk** [comment]
@DavidFFerreira since you added the `JZ-ZB-005` as `WP33-EU`, can you confirm this is correct? Seems it should be a `WP34-EU`? (https://github.com/Koenkk/zigbee-herdsman-converters/pull/2787)

**Comment by DavidFFerreira** [comment]
yup, pretty sure
![image](https://user-images.githubusercontent.com/32928749/167201994-289e0f88-6d67-44fe-86a0-d89985a06c17.png)
![image](https://user-images.githubusercontent.com/32928749/167202028-bb2ff933-547e-4ee2-878e-ca7f0c0346ec.png)

**Comment by DavidFFerreira** [comment]
@zandadoum did you order from lellki? Maybe a vendor mistake, but if it was I need to know

**Reply by zandadoum** [reply]
> @zandadoum did you order from lellki? Maybe a vendor mistake, but if it was I need to know

this was the process i followed:

1) went to https://www.zigbee2mqtt.io/supported-devices/
searching for a EU powerstrip with USB that can be toggled
found these LELLKI ones: 

![image](https://user-images.githubusercontent.com/51913999/167206274-e1eb6ac8-1bf7-4f83-93a3-ea1976846616.png)

and i chose the WP34-EU (it's not listed as super charging, which I don't care for)
as per pictures, the best way to differenciate the versions is to look at the design of the power button (round with LED circle for the WP34-EU)

2) checked where i could buy them. i ended up buying them in aliexpress on the "official lellki store" i made sure it was the WP34-EU

3) device arrived. correct model as per pictures and has this label on the back too:

![image](https://user-images.githubusercontent.com/51913999/167206651-41426944-3710-4169-88b5-4c4a2dc4bfbd.png)

so, yes. it's confirmed as WP34-EU

I added it to z2m and it works perfectly fine, but it is mis-identified as WP33-EU

**Comment by DavidFFerreira** [comment]
Ok @Koenkk  I did talk with vendor, and the WP34-EU and WP33-EU have the same chip model, soo, they are completly the same interiores without anything to diferenciate tham. Maybe I will propost to identify the model as WP33-EU/WP34-EU ?!

**Reply by Koenkk** [reply]
> Maybe I will propost to identify the model as WP33-EU/WP34-EU ?!

Agree, merged them now.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by DavidFFerreira** [comment]
@zandadoum @Koenkk  Thank you both
